vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+  skos:prefLabel "Traceability"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+  .
+dq-iso-25012:understandability
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Understandability"@en ;
+  skos:definition "The degree to which data has attributes that enable it to be read and interpreted by users, and are expressed in appropriate languages, symbols and units in a specific context of use. Some information about data understandability are provided by metadata."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Understandability"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+.
+dq-nsw:Dimensions
+  a skos:ConceptScheme ;
+  rdfs:comment "Find out more about the quality assurance processes from the NSW Government Standard for Data Quality Reporting. https://www.finance.nsw.gov.au/ict/resources/data-quality-standard"@en ;
+  rdfs:label "Dimensions of data quality (NSW OEH)"@en ;
+  skos:prefLabel "Dimensions of data quality (NSW OEH)"@en ;
+  rdfs:seeAlso <https://www.finance.nsw.gov.au/ict/resources/data-quality-standard> ;
+  dcterms:created "2020-11-16"^^xsd:date ;
+dcterms:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
+dcterms:modified "2020-11-16"^^xsd:date ;
+dcterms:publisher <http://linked.data.gov.au/def/gsq-alias/des> ;
+  skos:hasTopConcept dq-nsw:accessibility ;
+  skos:hasTopConcept dq-nsw:accuracy ;
+  skos:hasTopConcept dq-nsw:coherence ;
+  skos:hasTopConcept dq-nsw:institutional-environment ;
+  skos:hasTopConcept dq-nsw:interpretability ;
+.
+dq-nsw:accessibility
+  a dqv:Category ; a skos:Concept ;
+  rdfs:label "ACCESSIBILITY"@en ;
+  skos:prefLabel "Accessibility"@en ;
+  skos:inScheme dq-nsw:Dimensions ;
+.
+dq-nsw:accuracy
+  a dqv:Category ; a skos:Concept ;
+  rdfs:label "ACCURACY"@en ;
+  skos:prefLabel "Accuracy"@en ;
+  skos:inScheme dq-nsw:Dimensions ;
+.
+dq-nsw:assurance
+  a dqv:Dimension ; a skos:Concept ;
+  rdfs:label "Data assurance"@en ;
+  skos:prefLabel "Data assurance"@en ;
+  skos:definition "Data has been subject to a data assurance process (for example: Checking for errors at each stage of data collection and processing, or verifying data entry and making corrections if necessary.)"@en ;
+  skos:inScheme dq-nsw:Dimensions ;
+  dqv:inCategory dq-nsw:accuracy ;
+  skos:broader dq-nsw:accuracy ;
+.
+dq-nsw:authorized
+  a dqv:Dimension ; a skos:Concept ;
+  rdfs:label "Authorized data collection"@en ;
+  skos:prefLabel "Authorized data collection"@en ;
+  skos:definition "Data collection is authorised by law, regulation or agreement"@en ;
+  skos:inScheme dq-nsw:Dimensions ;
+  dqv:inCategory dq-nsw:institutional-environment ;
+  skos:broader dq-nsw:institutional-environment ;
+.
+dq-nsw:coherence

```

Comment:
No definition or documentation of meaning or examples.

Reply:
updated